src/components/DashboardLayout/DashboardHeader.jsx

```diff
@@ -105,8 +102,11 @@ function DashboardHeader(props) {
         </Box>
 
         <Stack direction="row" alignItems="center" gap="10px" pr="16px">
-          <Image
-            src={serviceLogoURL || placeholderService}
+          <img
+            src={
+              serviceLogoURL ||
+              "/assets/images/dashboard/service/servicePlaceholder.png"
+            }
             height={28}
             width={40}
             style={{ maxHeight: "28px", width: "auto", maxWidth: "180px" }}
```

Comment:
We show the org logo on the signup page. Did we remove the next/image from there as well?

Reply:
We were not using Next.js Image on the homepage